### PR TITLE
AAC Decoder tests

### PR DESCRIPTION
**Pull request by krzysztofciombor**
## Description
Adds tests for AAC Decoder. (Based on https://github.com/membraneframework/membrane-element-ffmpeg-h264 tests)
* Unit test for `Membrane.Element.AAC.Decoder.Native`
* Integration pipeline test for `Membrane.Element.AAC.Decoder`

Note: This depends on #1

### CODE REVIEW

File: test/decoder_test.exs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+defmodule DecoderTest do
+  import Membrane.Testing.Pipeline.Assertions
+  alias Membrane.Pipeline
+  use ExUnit.Case
```
Comment (Hajto) [inline test/decoder_test.exs]: Take a look at:
https://github.com/lexmag/elixir-style-guide#modules

We do not enforce it at all but in most modules we order them in similar way.

File: test/support/decoding_pipeline.ex [hunk 2]
```diff
@@ -0,0 +1,15 @@
+defmodule DecodingPipeline do
+  @moduledoc false
+
+  def get_options(%{in: in_path, out: out_path, pid: pid}) do
```
Comment (bblaszkow06) [inline test/support/decoding_pipeline.ex]: I assume it's some kind of leftover from pipeline that used to be here, there's no point to use map right now

Comment (bblaszkow06) [reply test/support/decoding_pipeline.ex]: `pid` - if changed to a normal function argument - can then have default `self()` value

File: test/decoder_test.exs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+defmodule DecoderTest do
+  use ExUnit.Case
+
+  import Membrane.Testing.Pipeline.Assertions
+
+  alias Membrane.Pipeline
+
+  def make_pipeline(in_path, out_path) do
+    Membrane.Testing.Pipeline.start_link(
```
Comment (bblaszkow06) [inline test/decoder_test.exs]: I would flatten that a bit and move `start_link` call to the `DecodingPipeline` module.
`make_pipeline` would be redundant then (if combined with changes in `DecodingPipeline` )

File: test/decoder_native_test.exs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    assert {:ok, decoded_frame} = Native.decode_frame(frame, decoder_ref)
+    assert {:error, :not_enough_bits} = Native.decode_frame(frame, decoder_ref)
+
+    assert {:ok, ref_file} = File.read(reference_path)
+
+    assert <<ref_frame::bytes-size(4096), _::binary>> = ref_file
+
+    assert bit_size(decoded_frame) == bit_size(ref_frame)
+    assert Membrane.Payload.to_binary(decoded_frame) == ref_frame
+  end
```
Comment (bblaszkow06) [inline test/decoder_native_test.exs]: I think you could add a scenario where the frame is passed in chunks and when more than one frame is passed at once.

**Pull request merged**